test/unit/imgix-provider.test.jsx

```diff
@@ -0,0 +1,109 @@
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+import ReactImgix from "../../src/index"
+import { ImgixProvider } from "../../es/HOCs/imgixProvider"
+
+const providerProps = {
+  domain: "sdk-test.imgix.net",
+  sizes: "100vw",
+}
+
+const imageProps = {
+  src: "https://assets.imgix.net/examples/pione.jpg",
+  sizes: "50vw",
+}
+
+describe('ImgixProvider', () => {
+
+  test('should not have context value defined if Provider has no props', () => {
+    const wrappedComponent = (
+      <ImgixProvider>
+        <ReactImgix {...imageProps}/>
+      </ImgixProvider>
+    )
+
+    const expectedProps = {
+      children: < ReactImgix src="https://assets.imgix.net/examples/pione.jpg" sizes="50vw" />,
+      value: {}
+    }
+
+    const renderedComponent = shallow(wrappedComponent)
+    expect(renderedComponent.props()).toEqual(expectedProps)
+  })
+
+  test('should set the context value to the Provider props', () => {
+    const wrappedComponent = (
+      <ImgixProvider {...providerProps}>
+        <ReactImgix {...imageProps}/>
+      </ImgixProvider>
+    )
+
+    // ensure Provider value correctly set
+    const expectedProps = {
+      children: < ReactImgix src="https://assets.imgix.net/examples/pione.jpg" sizes="50vw" />,
+      value: { domain: "sdk-test.imgix.net", sizes: "100vw", }
+    }
+
+    const renderedComponent = shallow(wrappedComponent)
+    expect(renderedComponent.props()).toEqual(expectedProps)
+  })
+
+  // TODO(luis): enable these tests once propMerger and propProcessor merged in
+  // test('should merge the Provider and Child props', () => {
+
+  //   const modifiedProps = { ...imageProps };
+  //   modifiedProps.src = "examples/pione.jpg"
+  //   modifiedProps.sizes = null
```

Comment:
```suggestion
  //   const modifiedProps = { 
  //     ...imageProps,
  //     src: "examples/pione.jpg",
  //     sizes: null,
  //   };
```

Reply:
This test won't work until the next PR in the sack is merged in. Going to remove it from this PR and instead add it back in whenever dependencies have been merged in.